File: 프로그래머스/unrated/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.java
class Solution {
    public int solution(int[] sides) {
        int min = 1001;
        for(int i=0; i<sides.length; i++){
            if(min>sides[i]){
                min = sides[i];
            }
        }
        int answer = ( 2 * min ) -1;
        return answer;
    }
}

File: 프로그래머스/0/120868. 삼각형의 완성조건 （2）/삼각형의 완성조건 （2）.js
function solution(sides) {
    let answer = 0;
    let maxLen = Math.max(sides[0],sides[1]);    
    let minLen = Math.min(sides[0],sides[1]);
    
    for(let i=maxLen+1; i<maxLen+minLen; i++){
            answer++;
    }
    
    
    for(let i=maxLen - minLen+1; i<=maxLen; i++){
            answer++;
    }
    return answer;
}